### ISSUE TITLE
Missing relative intensities in isotope-annotated records

### ISSUE THREAD

**Issue by schymane** [MassBank/RMassBank#140]
A dataset from Nikiforos passed MassBank validation but failed to upload - cause: missing relative intensities in the peaklist for 6 of 733 files. The peaks affected are mostly, but NOT all isotopes and one was nasty (an isotope, certainly an incorrect peak, much higher intensity than any others). 
ACCESSION: AU226005
  150.0667 C9H10O2+ 1 150.0675 -5.78
  150.0667 419020
ACCESSION: AU161902
  272.1274 C16H18NO3+ 1 272.1281 -2.6
  272.1274 34252 [this peak also appeared out of order]
ACCESSION: AU204904
  116.0071 C5H5[37]ClN+ 1 116.0076 -4.31
  128.0064 C6H5[37]ClN+ 1 128.0076 6.25
  116.0071 1780
  128.0064 16152
ACCESSION: AU203602
  147.0123 C3[13]CH5ClN3O+ 1 147.0137 -9.52
  147.0123 708
ACCESSION: AU160004 [the nasty case - for this one I deleted the peak in question to upload]
  296.0945 C17H15ClN3+ 1 296.0949 -1.35
  297.0981 C16[13]CH15ClN3+ 1 297.0979 0.67
  298.0919 C17H15[37]ClN3+ 1 298.0924 -1.68
  270.0787 693360 999 [for the record, the base peak]
  296.0945 72668 105
  297.0981 1320419
  298.0919 20596 30
ACCESSION: AU207104
  138.0744 C6[13]CH9N2O+ 1 138.0738 4.35
  138.0744 1340

@meowcat @ermueller (we're not quite sure where the cause is until we get the data from Nikiforos)

**Comment by meowcat** [comment]
If these should really be failpeaks that were added back, it might be an issue with where the relative intensities are recalculated? Don't know.

Anyway, were these done with the devel or with the old version?

**Comment by uchem-massbank** [comment]
MS$DATA_PROCESSING: WHOLE RMassBank 2.0.3
One of Erik’s github versions with the isotope labelling somewhere in between no and full refactoring. I think before - @ermueller can correct me.

**Comment by ermueller** [comment]
Yes, that's correct. Although I have no idea how this affects rel. int. stuff...

In any case, try again with the devel version and if anything goes wrong then I can take a closer look?

**Comment by schymane** [comment]
I think we need his *_RF.RData before we can figure out what went wrong where, I sent him an email already…I only have the records.

**Comment by schymane** [comment]
Missing relative intensities caused by manual intervention.
This will become a new issue to improve the isotope annotation... #142
